backend/src/routers/gameRouter.ts

```diff
@@ -360,7 +420,7 @@ const formatGames = async (games: any[], user: GammaUser | null) => {
 				}).length > 0,
 			ratingAvg: await getAverageRating(game.id),
 			ratingUser: user ? await getUserRating(game.id, user.cid) : null,
-			isPlayed: false
+			isPlayed: user ? (game.playStatus.filter((status: PlayStatus) => { return status.userId == uid })).length > 0 : false
 		}))
 	);
```

Comment:
This should be moved to the `gamesService` as we are querying the DB

Reply:
What do you mean? We aren't really querying, but rather using and formatting the data from the query?

Reply:
Sorry, I read wrong!